src/IssueViz/IssueVisualizationControl/IssueVisualizationControl.xaml

```diff
@@ -238,6 +239,162 @@
                 </DataTrigger>
             </Style.Triggers>
         </Style>
+
+        <!-- Flows Combo-box styles -->
+
+        <ControlTemplate x:Key="ComboBoxToggleButton" TargetType="ToggleButton">
```

Comment:
Is there anything I need to look at specifically in this template, or is it just a straight copy of the default template with the colours changed?

Reply:
Almost a copy paste, with irrelevant parts removed (i.e. disabled/editing state). I added style triggers for colors and ComboBoxItem template + visibility trigger.
```
<Style.Triggers>
    <DataTrigger Binding="{Binding CurrentIssue.Flows.Count}" Value="1">
        <Setter Property="Visibility" Value="Collapsed"/>
    </DataTrigger>
</Style.Triggers>
```
https://docs.microsoft.com/en-us/dotnet/desktop/wpf/controls/combobox-styles-and-templates?view=netframeworkdesktop-4.8

Reply:
I think it's worth commenting what is boilerplate and what isn't. It's not obvious and there's too much of it to follow easily.

I'd probably have put all of this in a separate resource dictionary in a separate file with appropriate comments and merged the resource dictionaries.